desktop/src/main/java/bisq/desktop/main/offer/offerbook/OfferBookListItem.java

```diff
@@ -79,7 +110,7 @@ public WitnessAgeData getWitnessAgeData(AccountAgeWitnessService accountAgeWitne
                     // either signed & limits lifted, or waiting for limits to be lifted
                     // Or banned
                     daysSinceSignedAsLong = TimeUnit.MILLISECONDS.toDays(optionalWitness.map(witness ->
-                            accountAgeWitnessService.getWitnessSignAge(witness, new Date()))
+                                    accountAgeWitnessService.getWitnessSignAge(witness, new Date()))
```

Comment:
Either, this is formatting of an older commit (which I have not found yet).  Or my IDE style rules have changed since I upgraded to latest version (doubtful because my bisq style rule file has not changed).

Lines 112-114 in the repo and after auto-formatting in my IDE are
```
                    daysSinceSignedAsLong = TimeUnit.MILLISECONDS.toDays(optionalWitness.map(witness ->
                                    accountAgeWitnessService.getWitnessSignAge(witness, new Date()))
                            .orElse(0L));

```